tasks/go.py

```diff
@@ -108,13 +109,20 @@ def deps(ctx, verbose=False):
     """
     Setup Go dependencies
     """
-
+    max_retry = 3
     print("downloading dependencies")
     with timed("go mod download"):
         verbosity = ' -x' if verbose else ''
         for mod in DEFAULT_MODULES.values():
             with ctx.cd(mod.full_path()):
-                ctx.run(f"go mod download{verbosity}")
+                for retry in range(max_retry):
+                    result = ctx.run(f"go mod download{verbosity}")
+                    if result.exited is None or result.exited > 0:
+                        wait = 10 ** (retry + 1)
```

Comment:
Question: Isn't it too long when retry == 2 ?

Reply:
that's the downside of the exponential retry. I was considering it was not that bad regarding the low risk of occurrence